src/test/resources/test-json-path.json

```diff
@@ -1458,5 +1458,163 @@
       },
       "otherKey": "***"
     }
+  },
+  {
+    "maskingConfig": {
+      "maskJsonPaths": [
+        "$.json.path"
+      ],
+      "maskJsonPathsAlternatively": [
```

Comment:
Maybe instead of using `maskJsonPathsAlternatively`, you could make it that `maskJsonPaths` could also accept a json object of format `"maskJsonPaths": ["a", { "jsonPaths": ["b", "c"], "maskNumbersWith": ...}`?

If that's to difficult, then I'd rather look into a specific test that tests such behavior for regular keys (don't remember the name, maybe CustomJsonMaskingTest or something like that)

Reply:
Should not be difficult, I will do that

Reply:
Implemented.